### ISSUE TITLE
Uninstall option

### ISSUE THREAD

**Issue by furicle** [atuinsh/atuin#1529]
While no one of sound mind would want to stop using atuin after they've figured out out, sometimes weird things happen, and you need to move on.

It would be nice if atuin offered an uninstall option, if run on a system where atuin is already present.

**Comment by ellie** [comment]
Hey!

> While no one of sound mind would want to stop using atuin

<3

The issue with doing the uninstall via Atuin, is that we have no idea how the user installed it. If installed via the system package manager, that is the route it will need to be uninstalled by. Our install script is literally just a switch that will use the system package manager wherever possible. 

It could maybe come with a switch to do the reverse, and uninstall instead, but the user will still need to edit their shell config by hand

**Comment by furicle** [comment]
Yes, just a switch to do the reverse I _think_ would be helpful.

Could something as simple as a grep though the shell config files for 'atuin' at least suggest to the user what they'd need to remove from their shell config as well?

I'm not familiar enough with all the possible config files to know just how effective that would be.

**Comment by ellie** [comment]
It really depends on how you installed it, Atuin is available in many system package managers. If you let me know

- how you installed it
- which system you use

Then I'd be able to suggest that for you.

More here: https://docs.atuin.sh/uninstall/

> cause of unpredictable behavior and lot of terminal weird behaviour only on systems where i installed and used it.

Are you able to elaborate?
